configs/config/pretrain/simclr/simclr_1node_resnet_imagenette_160.yaml

```diff
@@ -0,0 +1,111 @@
+# @package _global_
```

Comment:
is it possible to move this to `configs/config/pretrain/simclr/datasets/imagenette_160.yaml` and keep only the relevant changes in the yaml file?

Reply:
As discussed together, the idea of Imagenette is to help quickly iterate and debug issues. This is not really for pretraining not benchmarking in the real world, so I will move them to "debugging/pretrain/" and "debugging/benchmark/" as suggested.